### ISSUE TITLE
Scopes section of PAT page in settings has no accessibility labels

### ISSUE THREAD

**Issue by BrailleBennett** [modrinth/code#3798]
### Please confirm the following.

- [x] I checked the [existing issues](https://github.com/modrinth/code/issues?q=is%3Aissue) for duplicate problems
- [x] I have tried resolving the issue using the [support portal](https://support.modrinth.com)

### What browsers are you seeing the problem on?

Firefox, Safari

### Describe the bug

On the personal access token section in settings, when going to create or edit the scopes on a token, the checkboxes where you can choose which things that the token can do, there are no accessibility labels on it. They just appear as like 30 blank checkboxes to a screen reader, with 0 way to distinguish which one is which permission.

### Steps to reproduce

1. Turn on a screen reader (NVDA is easiest)
2. Navigate to the settings of your Modrinth account
3. Find the personal access token section under developer
4. Click create token
5. Try selecting scopes on your create screen
6. Observe how the screen reader simply announces "checkbox" with no labels about what each checkbox does

### Expected behavior

The checkboxes would be labeled with the permission it grants the token

**Comment by Hallskii** [comment]
Hi there, unfortunately most of the site don't meet accessibility standards and its something we want to improve in the future. It unfortunately isn't a priority right now so closing
